circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   java:
-    version: oraclejdk7
+    version: oraclejdk8
```

Comment:
with oraclejdk7 circle complains like this:

```
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._localedata.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._sunec.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._sunjce_provider.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._sunpkcs11.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._zipfs.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._dnsns.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._localedata.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._sunec.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._sunjce_provider.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._sunpkcs11.jar; error in opening zip file
error: error reading /usr/lib/jvm/jdk1.7.0/jre/lib/ext/._zipfs.jar; error in opening zip file
```

Java 8 doesn't have that problem. I'm not entirely sure where that problem comes from unfortunately :(